packages/apidom-ns-workflows-1/src/predicates.ts

```diff
@@ -14,6 +14,7 @@ import CriterionElement from './elements/Criterion';
 import ReferenceElement from './elements/Reference';
 import JSONSchemaElement from './elements/JSONSchema';
 // NCE types
+import WorkflowWorkflowsElement from './elements/nces/Workflows';
```

Comment:
This naming indicates that the `WorkflowWorkflowsElement` actually is a `workflows` field in `Workflow Object`. Which is not what this is trying to express.

```suggestion
import WorkflowsElement from './elements/nces/Workflows';
```

Reply:
My change indicates that it's an NCE for root level field called `workflows`.